components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -519,6 +519,7 @@ enum OnRampProvider {
   kWyre = 0,
   kRamp = 1,
   kSardine = 2,
+  kTransfero = 3,
```

Comment:
This collides with the value for Transak: https://github.com/brave/brave-core/pull/15948/files#diff-9f2cf11d57c8efee5461b92cdb38fa327b70c25fd8d009f7692049e8e88944a8R528